### PR TITLE
Translations take 2

### PR DESCRIPTION
**Pull request by mnoack**

### CODE REVIEW

File: spec/translation_spec.rb [hunk 1]
```diff
@@ -27,6 +27,29 @@ def to_s
   specify { expect(subject.to_s).to eq 'My name is Ruby' }
   specify { expect(class_translation).to eq 'My name is Class' }
 
+  specify {
+    allow(TranslationModel).to receive(:base_translation_class).and_call_original
```
Comment (alxberardi) [inline spec/translation_spec.rb]: For consistency, these two lines setting up the context should be in a before block within a context

**Comment by alxberardi** [comment]
👍 1 minor comment

**Comment by alxberardi** [comment]
Travis is unhappy now :(

**Comment by alvinypyim** [comment]
👍 Please fix travis before merging thanks!

**Pull request merged**